draft-irtf-cfrg-vdaf.md

```diff
@@ -5695,54 +5701,124 @@ useful feedback on and contributions to the spec.
 # Test Vectors {#test-vectors}
 {:numbered="false"}
 
-(TO BE REMOVED BY RFC EDITOR: machine-readable test vectors can be found at
-https://github.com/cfrg/draft-irtf-cfrg-vdaf/tree/main/poc/test_vec.)
+> TODO Update the reference in {{TestVectors}} by replacing `<TODO>` with
+> the commit hash with the final version of the test vectors.)
 
-Test vectors cover the generation of input shares and the conversion of input
-shares into output shares. Vectors specify the verification key, measurements,
-aggregation parameter, and any parameters needed to construct the VDAF. (For
-example, for `Prio3Sum`, the user specifies the number of bits for representing
-each summand.)
+The test vectors are available at {{TestVectors}}. The directory contains a set
+of JSON files. Each file contains a test vector for an instance of `Vdaf`
+({{vdaf}}). A test vector covers sharding, preparation, aggregation, and
+unsharding of each of several measurements. The test vector schema is defined
+below.
 
-Byte strings are encoded in hexadecimal. To make the tests deterministic, the
-random inputs of randomized algorithms were fixed to the byte sequence starting
-with `0`, incrementing by `1`, and wrapping at `256`:
+## Schema
+{:numbered="false"}
 
-~~~
-0, 1, 2, ..., 255, 0, 1, 2, ...
-~~~
+`verify_ke"`:
+: The verification key encoded in hexadecimal.
+
+`agg_param`:
+: The aggregation parameter of type `Vdaf.AggParam`.
+
+`prep`:
+: A list of objects with the following schema:
+
+    `measurement`:
+    : The measurement of type `Vdaf.Measurement`.
+
+    `nonce`:
+    : The nonce encoded in hexadecimal.
+
+    `rand`:
+    : The sharding randomness encoded in hexadecimal.
+
+    `public_share`:
+    : The expected public share encoded in hexadecimal.
+
+    `input_shares`:
+    : The expected list of input shares, each incoded in hexadecimal.
+
+    `prep_shares`:
+    : The expected list of prep shares generated by each Aggregator at each
+      round of preparation, encoded in hexadecimal.
+
+    `prep_messages`:
+    : The expected list of prep messages for each round of preparation, encoded
+      in hexadecimal.
+
+    `out_shares`:
+    : The expected list of output shares, encoded in hexadecimal.
+
+`agg_shares`:
+: The expected aggregate shares encoded in hexadecimal.
+
+`agg_result``
```

Comment:
nit: typo
```suggestion
`agg_result`:
```

Reply:
Done, and fixed a misspelled word below.